### DISCUSSION TITLE
Ajouter les pourcentages dans la répartition des cotisations

### DISCUSSION THREAD

**Discussion by johangirod** [betagouv/mon-entreprise#2653] [Idées]
En plus des valeurs absolues, il pourrait être intéressant d'avoir les pourcentages des différentes cotisations.
![image](https://user-images.githubusercontent.com/1775934/123275813-f01dc380-d504-11eb-9a6b-fd573bd5400e.png)

**Comment by mquandalle** [comment]
La remarque utilisateur a été fait sur le simulateur artiste-auteur qui ne dispose pas de la barre de répartition du revenu (car on ne calcule pas l'IR dessus).

![image](https://user-images.githubusercontent.com/1730702/124479425-8d4ee680-dda6-11eb-8a8c-0d158aed9821.png)

Je pense qu'afficher cette barre sur le simulateur artiste-auteur répondrait au besoin.

Il y a fort longtemps quand je suis arrivé sur le projet, je voulais afficher plus souvent des taux dans l'UI #546, mais le problème c'est qu'en affichant juste un taux on ne sait jamais trop à quelle assiette il correspond. La barre de répartition globale me semble donner une image plus pertinente que les taux “brut” auxquels ils faut ré-intégrer des abattements d'assiette ou au contraire des éléments non déductibles type CSG.

**Comment by johangirod** [comment]
Je pensais à afficher la part de chaque cotisation par rapport au total des cotisations (et non pas par rapport à l'assiette) . Cela est certes une donnée redondante vu qu'elle est déjà disponible sous la forme graphique, mais on ne se rend pas forcément compte par exemple que pour le cas d'un SMIC, la cotisation retraite équivaut à quasiment 50% du montant total des cotisations. 

Ceci étant dit, ce n'est pas une information très importante, elle pourrait être mise au second plan, par exemple au hover de la barre du graphique.

**Comment by mquandalle** [comment]
Ah oui, la proportion de chaque cotisation c'est pertinent !
